open-company/open-company-web
# Font size spacing tweaks

Card: https://trello.com/c/rgk3c72Z

To test:
- check font size and line height of title and abstract/body on FoC
- check font size and line height of title, abstract and body in expanded post
- check font size and line height of title and body in QP
- check font size and line height of title, abstract and body in FP
- check font size and line height of title, abstract and body on secure post view
- check all other items in the trello card
- check face pile on Foc of a post with 0 comments
- check face pile on Foc of a post with 1 comment
- check face pile on Foc of a post with 2 comments
- check face pile on Foc of a post with 3+ comments

To test FoC footer:
- add a post with 1 reaction
- add a post with 1 reaction and 2 comments
- add a post with 1 attachment only (0 comments 0 reactions)
- add a post with 3 reactions and 1 comment
- add a post with 4 reactions and 1 comment
- add a post with 4 reactions, 1 comment and 2 attachments
- [x] make sure on desktop you see all the details in FoC
- [x] make sure on mobile you see all the details if there are no attachments or less than 4 reactions
- [x] make sure on mobile you see at most 3 reactions for those who have more
- [x] make sure you never see attachments on FoC for mobile
